stage0/deny.toml

```diff
@@ -19,5 +19,17 @@ wildcards = "allow"
 # List of allowed licenses.
 # For more detailed information see http://go/thirdpartylicenses.
 [licenses]
-allow = ["Apache-2.0", "MIT"]
+allow = [
+  "Apache-2.0",
+  "Apache-2.0 WITH LLVM-exception",
```

Comment:
I don't think this license is required, and a few of the others. It probably only needed "Unicode-DFS-2016" and "BSD-3-Clause" on top of what was already there. Adding unused licenses to the allow list will cause noisy warnings in cargo deny.

Reply:
Fair enough, I just copied everything from the root `deny.toml` file.